homeassistant/components/nextcloud/update.py

```diff
@@ -0,0 +1,52 @@
+"""Summary data from Nextcoud."""
+from __future__ import annotations
+
+from homeassistant.components.update import UpdateEntity, UpdateEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import NextcloudDataUpdateCoordinator
+from .entity import NextcloudEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the Nextcloud update entity."""
+    coordinator: NextcloudDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+    major = coordinator.data.get("system_version", "0").split(".")[0]
+    if int(major) < 28:
+        return
```

Comment:
```suggestion
    if coordinator.data.get("system_update_available") is None:
        return
```

it should be enough to check if the key `system_update_available` exists